### DISCUSSION TITLE
🎊 Wagtail approved for participation to GSoC in 2023

### DISCUSSION THREAD

**Discussion by thibaudcolas** [wagtail/gsoc#2] [Announcements]
It’s official! We’ve been approved by Google to participate in the programme this year, alongside [172 other organisations](https://summerofcode.withgoogle.com/programs/2023/organizations) 🎉 

This repository and Discussions board will slowly but surely get busy – come say hi, [introduce yourself](https://github.com/wagtail/gsoc/discussions/1), and have a look at our [contributors guide](https://wagtail.org/gsoc-guide).

**Comment by Ayeshashaik759** [comment]
Hello everyone, I have just gone through the idea list of wagtail and I found the project Dark theme for Wagtail admin very interesting as it suits my tech stack. looking forward to contribute to this project any suggestion/guidance to start my contribution to this project will be appreciated.

**Reply by thibaudcolas** [reply]
Welcome @Ayeshashaik759! Unfortunately we’ve had to pull this project idea as we deemed it wasn’t substantial enough to be covered as a GSoC project. Please have a read through our [contributor guide](https://github.com/wagtail/gsoc/blob/main/contributor-guide.md) if the other project ideas interest you.

**Comment by salty-ivy** [comment]
Hello my fellow contributors, mentors and core team this is  `Aman Pandey`  AKA `salty-ivy` been contributing to wagtail for few months (since november 2022..)  really loved the experience and delighted to see wagtail in GSoc 2023 as well.

**Reply by thibaudcolas** [reply]
Welcome @salty-ivy! Great to see you here!

**Comment by Theskyspace** [comment]
Woah! I was using the product recently and found it extremely exciting; looking forward to contributing back to the community.

**Comment by dharmendra-byte** [comment]
Hello everyone, I have just gone through the idea list of wagtail and I found the project some ideas .ooking forward to contribute to this project any suggestion/guidance to start my contribution to this project will be appreciated.

**Reply by thibaudcolas** [reply]
@ProgrammingPirates it’s a bit late for that! Applications are closed as of 2 hours ago.

**Comment by dharmendra-byte** [comment]
But sir, I had submitted the proposal on time.
